src/views/Timer.vue

```diff
@@ -1,18 +1,30 @@
 <template>
   <div class="timers">
     <b-container fluid>
-      <AlertPanel v-if="this.$store.getters.componentState.alerts.state"/>
+      <b-row>
+        <AlertPanel v-if="this.$store.getters.componentState.alerts.state" :alerts="this.$store.getters.worldstate.alerts"/>
+        <TimePanel :time="this.$store.getters.worldstate.earthCycle"  :location="'Earth'" v-if="this.$store.getters.componentState.earth.state"/>
```

Comment:
`v-if` first so it's the same order on all the panels so it's easier to tell they're all using the same setup